docs/specifications/tests/DNSSEC-TP/dnssec13.md

```diff
@@ -92,73 +91,70 @@ message. The argument names are defined in the [argument list].
    3. Name server IP address and associated DNSKEY algorithm
       ("Algo not signed NS").
 
-6. For each name server IP in the NS IP set do:
-
-   1. Send *DNSKEY Query* over UDP.
-      1. If no DNS response is returned, then go to next name server IP.
-      2. Else, if the RCODE of the DNS response is not "NOERROR" go to next name
-         server IP.
-      3. Else, if the AA flag of the response is unset, then go to the next name
-         server IP.
-      4. Else, if the DNS response contains no DNSKEY record in the answer
-         section, then go to the next name server IP.
-      3. Else, if the DNS response contains no RRSIG for the DNSKEY RRset, then
-         go top the next name server IP.
-      3. Else, do:
-         1. Extract all DNSKEY records.
-         2. Extract the algorithm numbers from each DNSKEY record and add it to
-            a set ("DNSKEY Algorithm").
-         3. Extract all RRSIG records for the DNSKEY RRset from the response.
-         4. For each algorithm in *DNSKEY Algorithm* do:
-            1. If there is no RRSIG for the DNSKEY RRset created by
-               the algorithm then add name server IP and DNSKEY algorithm to the
-               *Algo not signed DNSKEY* set.
-
-   2. Send *SOA Query* over UDP.
-      1. If no DNS response is returned, then go to next name server IP.
-      2. Else, if the RCODE of the DNS response is not "NOERROR" go to next name
-         server IP.
-      3. Else, if the AA flag of the response is unset, then go to the next name
-         server IP.
-      4. Else, if the DNS response contains no SOA record in the answer
-         section, then go to the next name server IP.
-      3. Else, if the DNS response contains no RRSIG for the SOA RRset, then
-         go top the next name server IP.
-      3. Else, do:
-         1. Extract all RRSIG records for the SOA RRset from the response.
-         5. For each algorithm in *DNSKEY Algorithm* do:
-            1. If there is no RRSIG for the SOA RRset created by
-               the algorithm then add name server IP and DNSKEY algorithm to the
+6. For each name server IP in the *NS IP* set do:
+
+   1. Create an empty set of DNSKEY algorithms "DNSKEY Algorithm" (for each name
+      server IP).
```

Comment:
We already are in the loop "For each name server IP", and from what I've seen so far, you usually put the name of the set inside parentheses. I would then suggest to modify this sentence into something like:
```
  1. Create an empty set of DNSKEY algorithms ("DNSKEY Algorithm").
```

Reply:
Fixed.